### PR TITLE
Fix/m2 3553 circular dependencies

### PR DESCRIPTION
**Pull request by alex-i-e**

### CODE REVIEW

File: src/modules/Builder/features/ActivitySettings/ScoresAndReports/ScoreContent/ScoreContent.tsx [hunk 1]
```diff
@@ -197,7 +197,7 @@ export const ScoreContent = ({
             const title = t('scoreConditional', {
               index: key + 1,
             });
-            const headerTitle = <Title title={title} name={conditional?.name} />;
+            const headerTitle = <Title title={title} reportFieldName={conditional?.name} />;
```
[the remark this answers is not in the document]
Comment (IhorNesterenko99) [reply src/modules/Builder/features/ActivitySettings/ScoresAndReports/ScoreContent/ScoreContent.tsx:200]: upd:
const headerTitle = <Title title={title} reportFieldName={conditionalName} />;

[the remark this answers is not in the document]
Comment (alex-i-e) [reply src/modules/Builder/features/ActivitySettings/ScoresAndReports/ScoreContent/ScoreContent.tsx]: done

File: src/modules/Dashboard/features/Respondents/Respondents.styles.ts [hunk 2]
```diff
@@ -1,4 +1,4 @@
-import { styled } from '@mui/system';
+import { styled } from '@mui/material';
```
Comment (muhurov) [inline src/modules/Dashboard/features/Respondents/Respondents.styles.ts:1]: please combine imports

File: src/modules/Library/components/Header/Header.styles.ts [hunk 3]
```diff
@@ -1,4 +1,4 @@
-import { styled } from '@mui/system';
+import { styled } from '@mui/material';
```
Comment (muhurov) [inline src/modules/Library/components/Header/Header.styles.ts:1]: please combine imports

File: src/shared/components/Uploader/Uploader.styles.ts [hunk 4]
```diff
@@ -1,5 +1,5 @@
 import { Box } from '@mui/material';
-import { styled } from '@mui/system';
+import { styled } from '@mui/material';
```
Comment (muhurov) [inline src/shared/components/Uploader/Uploader.styles.ts:2]: please combine imports

File: src/shared/layouts/BaseLayout/components/TopBar/AccountPanel/AccountPanel.styles.ts [hunk 5]
```diff
@@ -1,4 +1,4 @@
-import { styled } from '@mui/system';
+import { styled } from '@mui/material';
 import { Drawer, Box, Button } from '@mui/material';
```
Comment (muhurov) [inline src/shared/layouts/BaseLayout/components/TopBar/AccountPanel/AccountPanel.styles.ts:2]: please combine imports

**Review by muhurov — APPROVED** [review]

**Review by IhorNesterenko99 — APPROVED** [review]

**Review by nmialik — APPROVED** [review]

**Pull request merged**